src/ophyd_async/core/device_save_loader.py

```diff
@@ -1,23 +1,40 @@
 from enum import Enum
-from typing import Any, Dict, Generator, List, Optional, Sequence
+from functools import partial
+from typing import Any, Callable, Dict, Generator, List, Optional, Sequence, Union
 
 import numpy as np
 import numpy.typing as npt
 import yaml
 from bluesky.plan_stubs import abs_set, wait
 from bluesky.protocols import Location
 from bluesky.utils import Msg
+from epicscorelibs.ca.dbr import ca_array, ca_float, ca_int, ca_str
```

Comment:
This is the right solution, but in the wrong place. We need to make sure `epicscorelibs` imports are only within the `ophyd_async.epics` packages. 

How about a global (yuck) directory of representers that are added to on inport of packages?

Reply:
I think this needs a broader discussion on where stuff belongs. I don't like the fact that we have plans in ophyd_async.core, but definitely agree epics things shouldn't belong there in the long run. For the meantime, while ophyd_async is still in pre-release, I'd be keen to just merge this in anyway and come up with a better long term solution later...

Reply:
@coretl I suggest merging, making an issue and fixing later because of time constraints